codes/classical/properties/dual.yml

```diff
@@ -24,6 +24,7 @@ protection: 'The dual of an \([n,k,d] \) code
 
 notes:
   - 'See Ref. \cite{arXiv:1706.01694} for constructions of binary self-dual codes of length 60.'
+  - 'See \href{http://www.unilim.fr/pages_perso/philippe.gaborit/SD/index.html}{Tables of Self-Dual Codes} for an auxiliary database of self-dual codes over \(GF(2) \textit{Type One/Type Two}, \(GF(3), \(GF(4) \textit{Euclidean/Hermitian}, \(GF(5), and \(GF(7).' 
```

Comment:
\( ... \) are parentheses for latex input. So every time you write math, you need to enclose it in these parentheses. If you open and don't close them, you'll get an error. You've opened them 5 times and didn't close them above.